prime-router/docs/api/validate.yml

```diff
@@ -0,0 +1,86 @@
+openapi: 3.0.2
+info:
+  title: Prime ReportStream
+  description: A router of public health data from multiple senders and receivers
+  contact:
+    name: USDS at Centers for Disease Control and Prevention
+    url: https://reportstream.cdc.gov
+    email: reportstream@cdc.gov
+  version: 0.2.0-oas3
+
+paths:
+  # The validation endpoints are public endpoints for validating payloads of various formats.
+  /validate:
+    post:
+      summary: Validate a message using client information
+      parameters:
+        - in: header
+          name: client
+          description: The client.sender to validate against
+          required: true
+          schema:
+            type: string
+          example: simple_report.default
+      requestBody:
+        description: The message to validate
+        required: true
+        content:
+          text/csv:
+            schema:
+              type: string
+            example:
+              header1, header2
+
+              value1, value2
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/CDCgov/prime-reportstream/master/prime-router/docs/api/reports.yml#/components/schemas/Report'
+        '400':
+          description: Bad Request
+
+  /validate?schema={schema}&format={format}:
```

Comment:
```
Semantic error at paths./validate?schema={schema}&format={format}
Query strings in paths are not allowed.
```

I suspect we want to use `oneOf:` with the two different `parameters:` blocks intead.

Reply:
Hm I didn't get that error locally with my IDEA plugin. What are you using to generate this so I can try too? Will look into oneOf!

Reply:
Okay I see this in https://editor.swagger.io/. Will try to find a work around, but oneOf not working so far

Reply:
I use https://editor.swagger.io/

Reply:
Updated! Unfortunately, there is not a way to do what I want, the official openAPI docs have a section titled Parameter Dependencies [here](https://swagger.io/docs/specification/describing-parameters/). I followed their suggestion:

> OpenAPI 3.0 does not support parameter dependencies and mutually exclusive parameters. There is an open feature request at https://github.com/OAI/OpenAPI-Specification/issues/256. What you can do is document the restrictions in the parameter description and define the logic in the 400 Bad Request response.

In the future, I will make sure to validate swagger changes @ https://editor.swagger.io/